src/service_client/logger.py

```diff
@@ -88,20 +88,22 @@ def add_stream_handler(logger: logging.Logger):
     logger.addHandler(ch)
 
 
-def get_logger(name: str = ""):
-    logging.getLogger("requests").setLevel(logging.ERROR)
-    logging.getLogger("urllib3").setLevel(logging.ERROR)
+logger_name = os.environ.get("LOGGER_NAME", "")
+urlib3_logger = logging.getLogger("urllib3")
+urlib3_logger.handlers = [logging.NullHandler()]
 
-    logger = logging.getLogger(name)
-    logger.setLevel(logging.DEBUG)
+logging.getLogger("requests").setLevel(logging.ERROR)
+urlib3_logger.setLevel(logging.ERROR)
 
-    add_log_file_handler(logger, "test_infra.log")
-    add_stream_handler(logger)
-    return logger
+_logger = logging.getLogger(logger_name)
```

Comment:
what's the reason for working with ``_logger`` as a temporary variable?
please add a comment to explain that if there's any specific issue with the straight-forward solution